src/MaterialDesignControls/Implementations/CustomMaterial3/CustomDatePicker.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using Xamarin.Forms;
+
+namespace Plugin.MaterialDesignControls.Material3.Implementations
+{
+    public class CustomDatePicker : DatePicker, IBaseMaterialFieldControl
+    {
+        public TextAlignment HorizontalTextAlignment { get; set; }
+
+        private DateTime? customDate;
+
+        public DateTime? CustomDate
+        {
+            get { return this.customDate; }
+            set
+            {
+                this.customDate = value;
+                if (this.customDate.HasValue)
+                {
+                    base.Date = this.customDate.Value;
+                    EmptyDate = false;
+                }
+                else
+                    EmptyDate = true;
+            }
+        }
+
+        public DateTime InternalDateTime
+        {
+            get { return base.Date; }
+        }
+
+        public string Placeholder { get; set; }
```

Comment:
I think that we can remove Placeholder and PlaceholderColor properties from this control and from the renders, because now we are resolving the placeholder with a label over the control. Same we could do with the rest of the controls (Entry, Picker, DoublePicker, etc).